crt_portal/cts_forms/views.py

```diff
@@ -256,7 +256,14 @@ def get(self, request, id):
         contact_form = ContactEditForm(instance=report)
         details_form = ReportEditForm(instance=report)
 
-        output.update({'contact_form': contact_form, 'details_form': details_form})
+        output.update({
+            'contact_form': contact_form,
+            'details_form': details_form,
+        })
+        output.update({
```

Comment:
I think we can drop this, the `POST` dictionary should be empty here since this function only handles gets

Reply:
Tackled that here: https://github.com/usdoj-crt/crt-portal/pull/516/commits/1831934e978db62d3405088fb235ee07a2f792cc